apps/settings/js/panels/call/call_forwarding.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Make sure all operations would be executed one by one.
+ *
+ * @module call/call_forwarding
+ */
+define(function(require) {
+  'use strict';
+
+  var CallConstant = require('panels/call/call_constant');
+
+  /**
+   * @class CallForwarding
+   * @returns {CallForwarding}
+   */
+  var CallForwarding = function() {
+    this._cardIndex = 0;
```

Comment:
It would make more sense to pass the instance of mozMobileConnection because the module seems not require to be aware of the card index.